features/temp_transform.py

```diff
@@ -0,0 +1,6 @@
+from databaker.framework import *
```

Comment:
this file is a leftover from the previous approach, you can remove it.

Reply:
Removed